core/src/test/java/org/elasticsearch/common/util/BigArraysTests.java

```diff
@@ -37,8 +37,8 @@
 
 public class BigArraysTests extends ESSingleNodeTestCase {
 
-    public static BigArrays randombigArrays() {
-        final PageCacheRecycler recycler = randomBoolean() ? null : ESSingleNodeTestCase.getInstanceFromNode(PageCacheRecycler.class);
+    private BigArrays randombigArrays() {
```

Comment:
wou'd it make more sense to push this method up into ESSingleNodeTestCase instead of duplicating it N times?